### DISCUSSION TITLE
What does "exposed" mean in the component APIs of V3 docs, and how do you use something which is exposed?

### DISCUSSION THREAD

**Discussion by chrissyast** [vuetifyjs/vuetify#17054] [Q&A]
e.g.

![image](https://user-images.githubusercontent.com/31564153/229242307-c63baf78-718e-44c1-8f79-2237c81247be.png)

**Comment by KaelWD** [comment]
https://vuejs.org/guide/essentials/template-refs.html#ref-on-component

**Reply by chrissyast** [reply]
This link doesn't seem to contain anything which answers my question 🤷‍♂️

**Reply by KaelWD** [reply]
They're things you can access through a template ref.
